ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -20,6 +20,14 @@ + (void)addContentsOfFolderToManifest:(NSString *)folderPath
     }
     
     for (NSString *fileName in folderFiles) {
+#if TARGET_IPHONE_SIMULATOR
```

Comment:
Let's remove this #if guard. I think we should skip this file in all cases, because that's what the server does.